src/runtime/cuda/l2_cache_flush.cc

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
We don't like to include code from other projects in TVM unless it lives in the `3rdparty` directory. As this code is basically the same as https://github.com/NVIDIA/nvbench/blob/1a13a2e724b8aa8aee27649ac6878babb63862a6/nvbench/detail/l2flush.cuh, it should be moved into `3rdparty/nvbench` and the license header at the top should be changed to the one in the nvbench repository.

@areusch is this acceptable?

Reply:
@yzh119 Let us move to `3rdparty/nvbench` then we can include it from there

Reply:
Please see an example of compiler-rt header 
- original file(with fixes) https://github.com/apache/tvm/blob/main/3rdparty/compiler-rt/builtin_fp16.h
- mention license here https://github.com/apache/tvm/blob/main/LICENSE#L235
- add licenses file to this folder https://github.com/apache/tvm/blob/main/licenses/LICENSE.builtin_fp16.txt

Reply:
Thank @tkonolige @tqchen for the comments, I'll refactor the code to include nvbench.

Reply:
@tkonolige @tqchen  I have refactored the code and added license files correspondingly, let me know if they look good to you:)